util/src/test/resources/org/killbill/billing/util/ddl_test.sql

```diff
@@ -1,5 +1,126 @@
 /*! SET default_storage_engine=INNODB */;
 
+DROP TABLE IF EXISTS accounts;
```

Comment:
Nit: I realize you extracted those hardcoded DDL update from our tests, so this is already better, but fundamentally it sucks that we have to duplicate such tables into our test resources just because they are not available.